exercises/raindrops/metadata.toml

```diff
@@ -1,4 +1,4 @@
 title = "Raindrops"
-blurb = "Convert a number to a string, the content of which depends on the number's factors."
+blurb = "Convert a number into its corresponding raindrop sounds - Pling, Plang and Plong"
```

Comment:
Should the blurb be a complete sentence with a terminating period? We probably want to keep that consistent across exercises.